nomos-services/tracing/Cargo.toml

```diff
@@ -10,8 +10,8 @@ async-trait = "0.1"
 futures = "0.3"
 nomos-tracing = { path = "../../nomos-tracing" }
 opentelemetry = { version = "0.25" }
-opentelemetry-otlp = "0.25"
-opentelemetry_sdk = { version = "0.25", features = ["rt-tokio"] }
+opentelemetry-otlp = "0.26"
```

Comment:
I think these were updated because of previously added tokio version, lets better create a new PR if tracing can be updated, I think couple more crates like `opentelemtry` should also be updated.

Reply:
These were updated due to a constraint that `surrealdb` adds. It requires a newer version of `Tokio`, which conflicts with this `opentelemetry` version's requirements.